src/BlockTypes/Checkout.php

```diff
@@ -109,6 +111,30 @@ protected function render( $attributes, $content ) {
 			$content = str_replace( '</div>', $inner_blocks_html . '</div>', $content );
 		}
 
+		/**
+		 * Checkout i3 added inner blocks for Order summary.
+		 * We need to add them to Checkout i2 templates.
+		 * The order needs to match the order in which these blocks were registered.
+		 */
+		$coupons_enabled                 = wc_coupons_enabled();
+		$order_summary_with_inner_blocks = '$0
+			<div data-block-name="woocommerce/checkout-order-summary-cart-items-block" class="wp-block-woocommerce-checkout-order-summary-cart-items-block"></div>
+			<div data-block-name="woocommerce/checkout-order-summary-subtotal-block" class="wp-block-woocommerce-checkout-order-summary-subtotal-block"></div>
+			<div data-block-name="woocommerce/checkout-order-summary-fee-block" class="wp-block-woocommerce-checkout-order-summary-fee-block"></div>
+			<div data-block-name="woocommerce/checkout-order-summary-discount-block" class="wp-block-woocommerce-checkout-order-summary-discount-block"></div>' .
+			( $coupons_enabled ? '<div data-block-name="woocommerce/checkout-order-summary-coupon-form-block" class="wp-block-woocommerce-checkout-order-summary-coupon-form-block"></div>' : '' ) .
+			'<div data-block-name="woocommerce/checkout-order-summary-shipping-block" class="wp-block-woocommerce-checkout-order-summary-shipping-block"></div>
+			<div data-block-name="woocommerce/checkout-order-summary-taxes-block" class="wp-block-woocommerce-checkout-order-summary-taxes-block"></div>
```

Comment:
Do we need to do a check for if taxes are enabled here?

Reply:
Most definitely, but I don't trust myself to do this change. There is no issue if the wrapper appears when it shouldn't because it's handled on the CSS side, but we can add a ticket for this separately.

Reply:
We shouldn't check for taxes being enabled, not coupons as well. That logic lives inside the actual block.